backend/socialdistribution/serializers.py

```diff
@@ -0,0 +1,15 @@
+from backend.api.models import author, post
+from rest_framework import serializers
+
+##############################################
+# serializers is for serialize the model into json format
+# This format is acceptable for http
+##################################################
+
+
+# serializer for post model into json representation
+class PostSerializer(serializers.ModelSerializer):
+  class Meta:
+    model = post
+    fields = ['post_id', 'origin_post_url', 'author_id', 'title', 'description', 'content', 'contentType', 'visibility'
+    , 'unlisted', 'published', 'url']
```

Comment:
This looks weird, it should be a part of my previous commit in `master`. plz remove it, I have updated the file

Reply:
@PENGYUXIONG should he just merge master into his branch once u merge and that should fix?

Reply:
might be. The problem is, this file is already in master. I pushed it in yesterday... I think he might need to delete this file.